src/style.css

```diff
@@ -1,3 +1,61 @@
+:root {
+  --link-color: #00ACDC;
+  --header-height: 60px
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  height: var(--header-height);
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav_list{
+  padding: 0;
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.nav_line {
+  display: flex;
+  position: relative;
+  align-items: center;
+  height: var(--header-height);
+}
+
+.img_logo{
+  width: 40px;
+  height: 40px;
+}
```

Comment:
The class '.img_logo' should be named following BEM naming conventions. If '.img_logo' is an element of a block, it should be named accordingly, for example, '.block__logo' if it's part of a 'block' component.